### ISSUE TITLE
Another install issue

### ISSUE THREAD

**Issue by pihentagy** [noteflakes/lyp#42]
After installing curl:

```
i5 pihentagy # wget -qO- https://git.io/getlyp | bash
Detecting latest lyp version...
Downloading https://github.com/noteflakes/lyp/releases/download/v1.3.1/lyp-1.3.1-linux-x86_64.tar.gz
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
100   605    0   605    0     0   1030      0 --:--:-- --:--:-- --:--:--  1030
100 6735k  100 6735k    0     0  1710k      0  0:00:03  0:00:03 --:--:-- 2109k
Extracting /tmp/lyp-release-installer/release.tar.gz
Could not find rugged-0.23.0 in any of the sources
Run `bundle install` to install missing gems.
```

**Comment by ciconia** [comment]
I updated the release files to remove the `rugged` dependency. Please check again and let me know if it works.

**Comment by pihentagy** [comment]
I will test this at my home PC. The newer issue reported just now occurred in another machine.

**Comment by ciconia** [comment]
Reported as fixed in #43.

**Comment by pihentagy** [comment]
Yes, it worked on my home PC.
